pkg/features/feature.go

```diff
@@ -0,0 +1,134 @@
+package features
+
+import (
+	feat "k8s.io/apiserver/pkg/util/feature"
```

Comment:
Consider renaming the import or the usage of `feat` throughout this code to avoid collisions